test/test_api/test_project_api.py

```diff
@@ -1177,6 +1178,17 @@ def test_task_progress(self):
         error_msg = "A valid project must be used"
         assert res.status_code == 404, error_msg
 
+        # query for the count of all tasks in the propject
+        res = self.app.get('/api/project/1/taskprogress', follow_redirects=True, headers=headers)
+        assert res.status_code == 200
+
+        # query for the count of all task count using a specific filter
+        res = self.app.get('/api/project/1/taskprogress?Fruit=apple', follow_redirects=True, headers=headers)
+        assert res.status_code == 200
+
+        # query for the count of all task count using null keyword
+        res = self.app.get('/api/project/1/taskprogress?Fruit=null', follow_redirects=True, headers=headers)
```

Comment:
As a sanity check, could we also add test cases for mixed-case values of `NULL` and `Null`? Thanks!

Reply:
done!

Reply:
In order to test field with Null value and no field exist, you could add few more tasks to the tests that contains Fruits=Null and some of them without friuts field in task.info, and confirm the count value upon return in addition to  status 200.